app/full_text_search.py

```diff
@@ -0,0 +1,64 @@
+from const import PERSON_FULLTEXT_INDEX_NAME
+
+from google.appengine.api import search
+import re
```

Comment:
Sort imports in alphabetical order. See:
https://google-styleguide.googlecode.com/svn/trunk/pyguide.html#Imports_formatting